docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -43,6 +41,72 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
         type: s3
   ```
 
+### Secure Token Service (STS) Installation
+
+* Create an Object Storage secret with keys as follows:
+
+    ```console
+    kubectl create secret generic tempostack-dev-s3 \
+      --from-literal=bucket="<BUCKET_NAME>" \
+      --from-literal=region="<AWS_REGION>" \
+      --from-literal=role_arn="<ROLE ARN>"
+    ```
+
+  where `tempostack-dev-s3` is the secret name.
+
+* Create a custom AWS IAM Role associated with a trust relationship to the Tempo's Kubernetes ServiceAccount
+  
+  ```yaml
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+          "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_PROVIDER}"
+        },
+        "Action": "sts:AssumeRoleWithWebIdentity",
+        "Condition": {
+          "StringEquals": {
+            "${OIDC_PROVIDER}:sub": [
+              "system:serviceaccount:${TEMPOSTACK_NS}:tempo-${TEMPOSTACK_NAME}",
+              "system:serviceaccount:${TEMPOSTACK_NS}:tempo-${TEMPOSTACK_NAME}-query-frontend"
+           ]
+         }
+       }
+     }
+    ]
+  }
+  ```
+  
+* Create an AWS IAM role:
+
+  ```yaml
+  aws iam create-role \
+    --role-name "tempo-s3-access" \
+    --assume-role-policy-document "file:///tmp/trust.json" \
+    --query Role.Arn \
+    --output text
+  ```
+
+* Attach a specific policy that role:
```

Comment:
Attach a specific policy to that role.

Reply:
Suggest changing the bullet to a numbered step: 
```suggestion
1. Attach a specific policy that role:
```